thirdeye/thirdeye-frontend/app/styles/wrapper/styles.scss

```diff
@@ -188,6 +186,24 @@ body {
   &__note {
     padding: 5px;
   }
+
+  &__custom-label {
+    font-size: 14px;
+  }
+
+  // overriding bootstrap for create alert form
+  .panel-title {
+    font-size: 15px;
+    color: #369ece;
+    font-weight: 600;
+    letter-spacing: .1px;
+  }
+
+  // overriding power-select basic field style
+  .ember-power-select-trigger {
+    padding-top: 2px;
```

Comment:
I believe I already made a change to make the power select input consistent with our existing `input`. 
This might not be needed.